src/main/java/core/basesyntax/converter/StringToFruitTransactionConverter.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+public class StringToFruitTransactionConverter implements Function<String, FruitTransaction> {
+
+    @Override
+    public FruitTransaction apply(String s) {
+        String[] arr = s.split(",");
+        if (arr.length != 3) {
+            throw new RuntimeException("Cannot convert string [" + s + "] to FruitTransaction");
```

Comment:
Consider providing a more informative error message when throwing a `RuntimeException`. This will help in debugging and understanding the exact issue with the input string. For example, specify which part of the string is causing the issue.